VAGRANT.md

```diff
@@ -6,13 +6,7 @@ Using Vagrant for development
 Setup
 -----
 
-Setup Vagrant and its dependencies:
-
-1. Install Ruby: [http://www.ruby-lang.org/](http://www.ruby-lang.org/)
-3. Install VirtualBox: [http://www.virtualbox.org/](http://www.virtualbox.org/)
```

Comment:
I think people still need to install VirtualBox separately, since we're using it as the provider.